app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -107,20 +116,39 @@ class HomeViewModel(
   }
 
   /**
-   * Returns a [HomeItemViewModel] corresponding to the promoted stories to be displayed for this learner
-   * (see [PromotedStoryListViewModel]), or null if this profile does not have any promoted stories.
-   * Promoted stories are determined by any recent stories started by this profile.
+   * Returns a [HomeItemViewModel] corresponding to the promoted stories[PromotedStoryListViewModel] and Upcoming topics
+   * [ComingSoonTopicListViewModel]  to be displayed for this learner or null if this profile does not have any promoted stories.
```

Comment:
Can we change the context to recommended here instead of using keywords like promoted i was a bit confused at first seeing promoted and recommended both.

Reply:
I would defer this to @rt4914 and @BenHenning.

Reply:
Thanks @rt4914 for making this clear.